server/mailrepository/mailrepository-cassandra/src/main/java/org/apache/james/mailrepository/cassandra/CassandraMailRepository.java

```diff
@@ -202,6 +202,14 @@ public long size() throws MessagingException {
         return countDAO.getCount(url).join();
     }
 
+    @Override
```

Comment:
I'm surprised, without this method your code was not compiling.

Why are you asking us to review things that are not compiling?

Reply:
it was before this morning merge.

Reply:
Yes that is a rebase fixup!

Reply:
Ok, thanks